### ISSUE TITLE
Add creative commons license for rules/content

### ISSUE THREAD

**Issue by plof27** [Leafwing-Studios/fonts-of-power#171]
Good to be explicit here, and CC makes more sense than something like MIT/Apache or GPL

**Comment by alice-i-cecile** [comment]
CC-BY-SA I think?

**Comment by plof27** [comment]
I was thinking maybe BY-NC? I think BY-SA let's you sell unmodified copies, which Is rather avoid. Although it would be nice to let people sell their homebrew.

**Comment by alice-i-cecile** [comment]
Yeah I want people to be able to make and sell modified work, but not resell unmodified copies of our stuff.
